src/main/java/com/playonlinux/ui/impl/javafx/setupwindow/StepRepresentationLicence.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2015 SLAGMOLEN Raphaël
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package com.playonlinux.ui.impl.javafx.setupwindow;
+
+import static com.playonlinux.domain.Localisation.translate;
+
+import javafx.geometry.Insets;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.TextArea;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
+
+import com.playonlinux.common.messages.CancelerMessage;
+import com.playonlinux.common.messages.CancelerSynchroneousMessage;
+
+public class StepRepresentationLicence extends AbstractStepRepresentation {
```

Comment:
A licenceStep should not have a leftImage. 
A licenceStep should inheritate at least from AbstractStepRepresentationWithHeader, or even StepRepresentationMessage